### ISSUE TITLE
semi-permanent white trapezoid on screen

### ISSUE THREAD

**Issue by mlauss2** [luciusDXL/TheForceEngine#333]
This not always visible white trapezoid has started appearing:
![image](https://github.com/luciusDXL/TheForceEngine/assets/123231678/5f7b4807-20b1-4550-9154-65ed9c0e617d)

Could it be related to this error message?
```
[Error : Shader] Failed to compile vertex shader!

[Error : Shader] Error: 0:39(2): preprocessor error: syntax error, unexpected HASH_TOKEN
```

En/Disabling Captions doesn't change it; it's visible in the classic GPU renderer and truecolor renderer.

**Comment by luciusDXL** [comment]
Is the quad there if you disable bloom?

**Comment by luciusDXL** [comment]
Also, what GPU are you using?

**Comment by luciusDXL** [comment]
I improved the Shader error messages, so can you run again and post the updated error. It will at least point to the shader producing the issue now.

**Comment by mlauss2** [comment]
AMD 7900
it's not bloom related; its appearance depends on location and view vector.
(Video: http://mlau.at/20230925_192109.mp4 )

[Error : Shader] Failed to compile 'Shaders/gpu_render_modelSolid.vert'!
[Error : Shader] Error: 0:38(2): preprocessor error: syntax error, unexpected HASH_TOKEN
[Error : TextureGPU - OpenGL] Failed to create texture - size: (3440, 1440), format: TEX_RGBA8.

**Comment by mlauss2** [comment]
And another thing:  the red projectiles of weapons 2 and 3 do not render any more either.

I'll dig out my nvidia box to double check that this is not a mesa driver error.

**Comment by luciusDXL** [comment]
Ok, great. Thanks for the extra information. Yeah, this definitely looks like shader compile errors. Given its with solid models - it makes sense that the blaster bolts are no longer working for you.

**Comment by luciusDXL** [comment]
Can you do me a favor? Go to lighting.h and add a newline at the end. See if that solves the issue.

**Comment by mlauss2** [comment]
Yeah, that solved it :) Good catch!
The trapezoid is gone and projectiles render again.

**Comment by luciusDXL** [comment]
*Sigh* - Sometimes I hate GLSL compilers... Ok, I will submit a fix.

**Comment by mlauss2** [comment]
fixed by 17de2b9
